src/createServer.js

```diff
@@ -1,3 +1,40 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { findErrors } = require('./validation');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-type', 'application/json');
+
+    const newURL = new URL(req.url, 'http://localhost:5700');
```

Comment:
You can take this "localhost:5700' information from req.headers. But u have to add 'http://' by yourself.

Its good practise because if u want to run it on some custom domain u dont have change anything in code